#221 Introduced unused-imports warnings which lead to failures in GHC

I think #221 introduced some code which generates "unused-imports" warnings when compiled with GHC. This leads to the fact that the submodule within GHC cannot be updated, since -Werror is enabled. I observed this in the following pipeline: https://gitlab.haskell.org/ghc/ghc/-/jobs/1722073

```console
===> Command failed with error code: 1
_build\stage1\libraries\Win32\build\System\Win32\WindowsString\Console.hs:67:1: error: [GHC-66111] [-Wunused-imports, Werror=unused-imports]
    The import of `Graphics.Win32.Misc' is redundant
      except perhaps to import instances from `Graphics.Win32.Misc'
    To import instances alone, use: import Graphics.Win32.Misc()
   |
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
_build\stage1\libraries\Win32\build\System\Win32\WindowsString\Console.hs:68:1: error: [GHC-66111] [-Wunused-imports, Werror=unused-imports]
    The import of `Graphics.Win32.GDI.Types' is redundant
      except perhaps to import instances from `Graphics.Win32.GDI.Types'
    To import instances alone, use: import Graphics.Win32.GDI.Types()
   |
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_build\stage1\libraries\Win32\build\System\Win32\WindowsString\Console.hs:70:1: error: [GHC-66111] [-Wunused-imports, Werror=unused-imports]
    The import of `Foreign.C.Types' is redundant
      except perhaps to import instances from `Foreign.C.Types'
    To import instances alone, use: import Foreign.C.Types()
   |
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_build\stage1\libraries\Win32\build\System\Win32\WindowsString\Console.hs:71:1: error: [GHC-66111] [-Wunused-imports, Werror=unused-imports]
    The import of `Foreign.C.String' is redundant
      except perhaps to import instances from `Foreign.C.String'
    To import instances alone, use: import Foreign.C.String()
   |
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_build\stage1\libraries\Win32\build\System\Win32\WindowsString\Console.hs:72:1: error: [GHC-66111] [-Wunused-imports, Werror=unused-imports]
    The import of `Foreign.Ptr' is redundant
      except perhaps to import instances from `Foreign.Ptr'
    To import instances alone, use: import Foreign.Ptr()
   |
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
_build\stage1\libraries\Win32\build\System\Win32\WindowsString\Console.hs:74:42: error: [GHC-38856] [-Wunused-imports, Werror=unused-imports]
    The import of `pokeArray'
    from module `Foreign.Marshal.Array' is redundant
   |
   |                                          ^^^^^^^^^
```

--- a/System/Win32/WindowsString/Console.hsc
+++ b/System/Win32/WindowsString/Console.hsc
@@ -63,14 +63,9 @@ import System.Win32.WindowsString.Types
 import System.Win32.Console.Internal
 import System.Win32.Console hiding (getArgs, commandLineToArgv)
 import System.OsString.Windows
-import Graphics.Win32.Misc
-import Graphics.Win32.GDI.Types (COLORREF)
 
-import Foreign.C.Types (CInt(..))
-import Foreign.C.String (withCWString, CWString)
-import Foreign.Ptr (Ptr, plusPtr)
 import Foreign.Storable (Storable(..))
-import Foreign.Marshal.Array (peekArray, pokeArray)
+import Foreign.Marshal.Array (peekArray)
 import Foreign.Marshal.Alloc (alloca)
 
 

--- a/Win32.cabal
+++ b/Win32.cabal
@@ -1,3 +1,4 @@
+cabal-version:  2.0
 name:           Win32
 version:        2.13.4.0
 license:        BSD3
@@ -11,10 +12,10 @@ category:       System, Graphics
 synopsis:       A binding to Windows Win32 API.
 description:    This library contains direct bindings to the Windows Win32 APIs for Haskell.
 build-type:     Simple
-cabal-version:  2.0
 extra-source-files:
     include/diatemp.h include/dumpBMP.h include/ellipse.h include/errors.h
     include/Win32Aux.h include/win32debug.h include/alignment.h
+extra-doc-files:
     changelog.md
 
 flag os-string

